server/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy

```diff
@@ -114,4 +114,5 @@ grant codeBase "${codebase.netty-transport}" {
 
 grant {
   permission java.net.SocketPermission "127.0.0.1", "connect, resolve";
+  permission java.nio.file.LinkPermission "symbolic";
```

Comment:
hmm... should we grant this permission to server too? (or is it already granted or not needed?)

Reply:
I only need it for the test since it's the only place where I create a symlink (permission needed). The non-test code simply needs to resolve the symlink if the target data location is a symlink, no permission changes needed.